docs/rules/no-sequences.md

```diff
@@ -63,10 +63,48 @@ while ((val = foo(), val < 42));
 with ((doSomething(), val)) {}
 ```
 
+## Options
+
+This rule takes one option, an object, with the following properties:
+
+* `"allowInParentheses"`: If set to `false`, this rule disallows expression sequences that are explicitly wrapped in parentheses. Default value is `true`.
```

Comment:
Hm you are right. What do you think, should the option set to `false` also disallow parentheses in for loops at all, so all of these would fail?
```
for ((i = 0, j = 10); i < j; i++);
for (i = 0; (test, i < j); i++);
for (i = 0; i < j; (i++, j--));
```
That would be more consistent in my eyes. Sequences without commas still would be allowed of course.

Reply:
I'm not sure if it makes sense to specifically disallow parenthesized sequence expressions in positions where unparenthesized sequence expressions are allowed. I think the accepted proposal is to just disable the exception, so the option should probably just ignore those checks like it's already implemented in this PR.

Reply:
Ok, then I'll adjust the sentence as you suggested, and add these additional test cases to the "passing" part of the unit test.

Reply:
Good idea to add tests that confirm the intent!